.github/ISSUE_TEMPLATE/report-a-question-in-the-wetransform-forum-.md

```diff
@@ -0,0 +1,12 @@
+---
+name: 'Report a question in the wetransform Forum '
+about: On importing or transforming the data into hale»studio
+title: 'Question: '
+labels: question
+assignees: ''
+
+---
+
+If you have problems with importing or transforming the data into hale»studio please drop up a question here:
```

Comment:
If you have questions regarding the usage of hale»studio, please drop us a question here: